testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -20,27 +20,31 @@
 import java.net.UnknownHostException;
 import java.nio.ByteBuffer;
 import java.time.Duration;
+import java.util.Date;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Optional;
 import java.util.function.Supplier;
 
 import com.datastax.oss.driver.api.core.cql.BoundStatement;
 import com.datastax.oss.driver.api.core.cql.Statement;
+import org.junit.AfterClass;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.BeforeClass;
```

Comment:
Could we use only JUnit 5 annotations just for the sake of clarify and consistency? WDYT?